calcom/cal.com
# fix: Unpublished screens

## What does this PR do?

When an unpublished team or organization was accessed, a 404 error appeared, we now show an unpublished screen, whether is for that team or org, or a user/event-type inside a team, or a user/sub-team/event-type inside an org.

Also fixes a few cases where not the correct entity was fetched, as well as UI issues with some unpublished screens that existed already, see the following old version, not very well centered:

<kbd><img src="https://github.com/calcom/cal.com/assets/467258/8c740a37-26fc-44ca-b9b9-b1c0618d171a" /></kbd>

And now:

<kbd><img width="1422" alt="image" src="https://github.com/calcom/cal.com/assets/467258/3fc419ce-eeb4-460a-8f8c-b85cd47e0b11"></kbd>

And for teams, the entity is actually "team":

<kbd><img width="1422" alt="image" src="https://github.com/calcom/cal.com/assets/467258/065ce1fd-2e1b-414c-8183-96e276107e90"><kbd>

Fixes #10400 

## Type of change


## How should this be tested?

> **Note**
> E2E tests have been added as part of this PR

Unpublished entities were added to the seed, so visit the following URLs and see their corresponding unpublished screens:

- `http://localhost:3000/team/foobar`
- `http://localhost:3000/team/foobar/collective`
- `http://localhost:3000/org/acme` same as `http://acme.cal.local:3000`
- `http://localhost:3000/org/acme/team/marketing` same as `http://acme.cal.local:3000/marketing`
- `http://localhost:3000/org/acme/team/marketing/collective` same as `http://acme.cal.local:3000/marketing/collective`
- `http://localhost:3000/org/acme/john` same as `http://acme.cal.local:3000/john`
- `http://localhost:3000/org/acme/john/30min`same as `http://acme.cal.local:3000/john/30min`

## Mandatory Tasks
